llvm/include/llvm/Support/DebugCounter.h

```diff
@@ -53,6 +54,18 @@ namespace llvm {
 
 class raw_ostream;
 
+struct Chunk {
```

Comment:
"Chunk" is an extremely generic name for something to exist globally in the `llvm` namespace. Please change this to reduce the risk of name collisions. For example, you could make it a nested struct in `DebugCounter`, or rename it to `DebugCounterChunk`.

(And change `{print,parse}Chunks` function analogously.)